pandas/tools/tests/test_tile.py

```diff
@@ -288,7 +288,10 @@ def test_qcut_duplicates_bin(self):
 
     def test_single_bin(self):
         # issue 14652
-        expected = Series([0, 0])
+        # Explicit dtype since Series produces int64 for ints, while cut
+        # (due to numpy.searchsorted) would use int32 on i386, so let's assure
+        # correct default to the architecture int
```

Comment:
though this works fine on windows now (e.g. 64-bit points on 32-bit systems).

and are you on master? this looks like an older line number.